core/services/log/broadcaster_test.go

```diff
@@ -1201,32 +1197,39 @@ func TestBroadcaster_ProcessesLogsFromReorgsAndMissedHead(t *testing.T) {
 		log1R       = cltest.RawNewRoundLog(t, addr, blockHash1R, 1, 0, false)
 		log2R       = cltest.RawNewRoundLog(t, addr, blockHash2R, 2, 0, false)
 
-		head0 = models.Head{Hash: blockHash0, Number: 0}
-		// head1 - missing
```

Comment:
not sure what scenario this was covering - we should never have a missing head in the chain normally